docs/src/explicit/RKN.md

```diff
@@ -0,0 +1,30 @@
+# OrdinaryDiffEqRKN
+
+Second order solvers.
+
+```@eval
+first_steps = evalfile("./common_first_steps.jl")
+first_steps("OrdinaryDiffEqRKN", "Nystrom4")
```

Comment:
The generic lorenz example probably is not a good fit for second order?

Reply:
Yeah it needs a SecondOrderODEProblem, IIRC there's one in the test.